ted_sws/master_data_registry/model/rdf_fragment.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+
+# rdf_fragment.py
+# Date:  29.07.2022
+# Author: Stratulat Ștefan
+# Email: stefan.stratulat1997@gmail.com
+from typing import List, Tuple
+
+from rdflib.term import URIRef
+
+from ted_sws.core.model import PropertyBaseModel
+
+
+class RDFFragment(PropertyBaseModel):
```

Comment:
rdflib.Graph is already an RDFFragment. 

Please do not create this class unless you need it for other purposes, which may be. But NOT for storing a list of triplets.